pkg/cim/volume.go

```diff
@@ -0,0 +1,241 @@
+package cim
```

Comment:
I believe only methods from this file are used in pkg/os/volume/api.go. Would it be possible to add the other files in pkg/cim in a different PR?

Reply:
The functions in `cim` package are grouped by WMI classes.
disk.go and volume.go with required functions are kept for this Volume API PR.
I will do a rebase on the other PRs.